### DISCUSSION TITLE
Emissive textures in plan?

### DISCUSSION THREAD

**Discussion by Fuzlop** [IrisShaders/Iris#872] [Q&A] [answered]
Hi, I want to know if you're going to implement equivalent of emissive texture feature of Optifine
Edit: Sorry I mean enabling emissive textures from custom texture packs. Using _e files.

**Comment by BeansBeefBroccoli** [comment]
This is a good idea, but it would probably be implemented in another mod, not Iris. Even if Iris did, they wouldn't do it before they reach most of their [goals](https://github.com/IrisShaders/Iris#goals).

**Comment by user859347** [comment, marked as answer]
You can use the mod "Puzzle" for emissive mob textures, however another mod named Continuity will add the rest at some point later.

Github: https://github.com/PuzzleMC/Puzzle
Modrinth: https://modrinth.com/mod/puzzle
Curseforge: https://www.curseforge.com/minecraft/mc-mods/puzzle

**Comment by OttCS** [comment]
A lot of shaders actually have "emissive" texture rendering, like the emissive ores in [GravityShade](https://gravityshade.netlify.app/) and Complementary. Using a shader that has emissive rendering for the blocks you want probably is your best bet, just add the block ids to the block.properties file.

**Comment by teddyxlandlee** [comment]
~~Why can't we __merge__ the implementations of these mods into an "__Iris+ submod__"?
I'm not sure if mods like __Puzzle__ are 95%+ compatible with Iris, but a __submod__ of Iris can.~~  
![image](https://user-images.githubusercontent.com/67260053/151979287-af98f9f6-6827-4342-99fc-8882549784e7.png)  
All right, aha
